rhoas/provider.go

```diff
@@ -68,13 +69,26 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
 	// Warning or errors can be collected in a slice type
 	var diags diag.Diagnostics
 
+	cfg := config.NewFile()
+
+	loc, err := cfg.Location()
```

Comment:
This is good fix. We going to move to different location in next version after @craicoverflow PR is released: https://github.com/redhat-developer/app-services-cli/pull/632